src/assets/icons/notifications.svg

```diff
@@ -1,4 +1,4 @@
-<!-- https://material.io/icons/#ic_notifications -->
```

Comment:
see this [comment](https://github.com/cloudfour/cloudfour.com-patterns/pull/703/files#r425411470)